plugins/techdocs/src/EntityPageDocs.tsx

```diff
@@ -17,13 +17,21 @@
 import React from 'react';
 import { Entity } from '@backstage/catalog-model';
 import { Reader } from './reader';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
 
 export const EntityPageDocs = ({ entity }: { entity: Entity }) => {
+  // Lower-case entity triplets by default, but allow override.
+  const toLowerMaybe = useApi(configApiRef).getOptionalBoolean(
+    'techdocs.legacyUseCaseSensitiveTripletPaths',
+  )
+    ? (str: string) => str
+    : (str: string) => str.toLocaleLowerCase('en-US');
```

Comment:
Now when we have a third duplicate of this implemented, can we add this as a helper function at `/src` and use the same across the EntityPageDocs, [DocsCardGrid](https://github.com/backstage/backstage/blob/7439368ce532e98bea8994b2be1899c9af9626ca/plugins/techdocs/src/home/components/DocsCardGrid.tsx) and [DocsTable](https://github.com/backstage/backstage/blob/7439368ce532e98bea8994b2be1899c9af9626ca/plugins/techdocs/src/home/components/DocsTable.tsx)

Reply:
done that emma :)